sass/includes/_content-warning.scss

```diff
@@ -1,26 +1,56 @@
 .content-warning {
-  display: flex;
-  background-color: $color__yellow_3;
-  border-left: 4px solid $color__yellow_2;
-  padding: 1.25rem 2rem;
-  margin-top: 2rem;
-
-  p {
-    @include font-size(xs);
-    color: $color__black;
-    margin-bottom: 0;
+
+  background-color: $color__yellow_6;
+  border-left: 4px solid #FFCC00;
+  padding: 24px 28px;
+
+  @media screen and (min-width: $screen__sm) {
+    display: flex;
+    padding: 1.25rem 2rem;
+    margin-top: 2rem;
+  }
+
+  @media screen and (min-width: $screen__md) {
+    margin-bottom: 3rem;
+  }
+
+  &--normalise-margin {
+    // normalise full-wdith mobile display
+    margin-left: -15px;
+    margin-right: -15px;
+  }
+
+  &--normalise-article-margin {
```

Comment:
Note: Both of these are work-arounds for the current widths of heros/ body widths. You can see I'm trying to normalise the content warning to be full-width to the page as best as we can.